### ISSUE TITLE
Built-in video recorder

### ISSUE THREAD

**Issue by wyskoj** [wyskoj/midis2jam2#100]

**Comment by wyskoj** [comment]
The implementation of this is complete, but I can't get it to work after being packaged with install4j. Will need to investigate it futher.

**Comment by steviator** [comment]
Did you ever manage to sort this out? Am I just too dumb to find the setting - entirely possible, I might be the world's least observant person.

**Comment by wyskoj** [comment]
@steviator No, I never got around to it. Using an external tool like OBS gets the job done.

**Comment by ViratS-best** [comment]
Bro please! We need this!
